src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -28,7 +28,7 @@ description: "Set up Azure Active Directory Single Sign-On (SSO) on Sentry."
 
 1. For Section (1), labeled "Basic SAML Configuration", enter the following data in each line and save your changes.
 
-   - Identifier (Entity ID): `https://sentry.io/saml/metadata/YOUR_ORG_SLUG/`
+   - Identifier (Entity ID): `https://sentry.io/saml/metadata/YOUR_ORG_SLUG/` <Alert level="warning">Ensure that the URL includes a trailing slash to prevent Azure from throwing a misconfiguration error (AADSTS650056).</Alert>
```

Comment:
Are we able to update our side of the integration to always add the slash?

Reply:
Unfortunately not. This config lives on Azure as a ID and our app has to send a string that matches against it. 

We could try the validation twice (once with the slash, once without) but IMO this is an easier/faster fix to take some load off the support team.